Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1422,6 +1422,678 @@ bool RenderStyle::diffRequiresLayerRepaint(const RenderStyle& style, bool isComp
     return false;
 }
 
+void RenderStyle::collectAffectedProperties(const RenderStyle& other, CSSPropertiesBitSet& changingProperties) const
+{
+    // This function conservatively answsers two things,
+    //   1. Whether we can use CSSTransitionAll fast path.
+    //   2. What CSS properties we should visit in the fast path.
```

Comment:
Now, this patch covers all the cases, and removed the fast path. So (1) should be removed, it is always handled in this code.

> This really does seem like something that should be autogenerated from CSSProperties.json. It will be difficult to keep it up to date when new properties are added.

It is ideal. But right now, we do not have a mechanism to generate data structures, and `RenderStyle::diff` is also not doing it. So for now, this patch is aligning the approach to `RenderStyle::diff`.